package.json

```diff
@@ -42,6 +42,7 @@
 		"@types/firefox-webext-browser": "^65.0.0",
 		"@types/jsdom": "^12.2.1",
 		"@types/onetime": "^2.0.0",
+		"@types/react": "^16.8.4",
```

Comment:
I don't think we have any choice but to include the React types for JSX, or dom-chef needs to bring them in as a dependency (probably makes more sense as it's for dom-chef).

Reply:
Yeah, I think `dom-chef` should ship those.